apps/jobs/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.conf.urls import url
-# from .views import ()
 
+from . import views
 
 urlpatterns = [
+    url(r'challenge/(?P<challange_id>[0-9]+)/'
+        r'challenge_phase/(?P<challange_phase_id>[0-9]+)/',
```

Comment:
Okay, but it still feels odd to not have a submission end_point appended after challenge_phase_id. The post request should be sent to /challenge/<challenge_id>/challenge_phase<phase_id>/submission right?